### PR TITLE
[UI] Avoiding overlap of anchor with link on search result page 

### LINKED ISSUE #5917: [UI] Overlapping of icons on search result page
#### Current Behavior
On hovering over the bullet point of the list of results shown in search results [page](https://docs.meshery.io/search/) one extra attachment icon is coming and it's overlapping with the bullet points. 

#### Desired Behavior
Either the attachment icon can be removed and just showing the underline border below the heading of the list item would look sleek, or we can add some spacing between the attachment icon and bullet points so that they don't overlap. 

#### Screenshots/Logs
Mashery_1

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by kamalbuilds**
Signed-off-by: Kamal Nayan <95926324+legendarykamal@users.noreply.github.com>

**Description**

This PR fixes #5917 

**Notes for Reviewers**
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 704cc26b72b9f8f01a0569daccd6efe4aef76833 at: https://62f88dcb6ae19332b8f6dd1b--meshery-docs-preview.netlify.app

File: docs/assets/css/anchorjs.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
 .anchorjs-link {
     color: inherit !important;
     text-decoration: none !important; /* do not underline */
+    margin-left: -1.6em; /* inorder to avoid overlap with the icon */
```
Comment (Kanika637) [inline docs/assets/css/anchorjs.css:8]: A little correction, it should be "-1.6 rem"

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: docs/assets/css/anchorjs.css [hunk 2]
```diff
@@ -23,6 +24,7 @@
     
     *:hover > .anchorjs-link:hover,
     .anchorjs-link:focus {
+    margin-left: -1.6em;
```
Comment (Kanika637) [inline docs/assets/css/anchorjs.css:27]: It should be "-1.6 rem"

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Screenshot 2022-08-16 at 2 51 46 AM
I can still see the overlap

**Comment by l5io** [comment]
🚀 Preview for commit 5827685852fdc3fedb289090910684c803e2485b at: https://62fb6c7de701a500759a6175--meshery-docs-preview.netlify.app

**Comment by legendarykamal** [comment]
@Kanika637 @warunicorn19 I have applied changes in the css file but they are not reflecting back in the docs, am i missing something?
Also I have tried making other simple changes but they too dont get reflected back when i see!

**Comment by warunicorn19** [comment]
I'm not too sure, there might be multiple CSS files. You could bring this up on the Meshery Dev call today to discuss if you'd like.

**Comment by legendarykamal** [comment]
Hey @nishi6verma happy that you are interested 
Go ahead ⏩
Note - If the changes are small, you can request them in this open PR
But If they are more and related to other files, then you can put up a new fresh PR for it.

**Pull request closed without merging**